src/sdp.ts

```diff
@@ -1,109 +1,119 @@
-import {inappropriateMultiaddr, invalidArgument, unsupportedHashAlgorithm} from './error.js';
-import {logger} from '@libp2p/logger';
-import {Multiaddr} from '@multiformats/multiaddr';
-import * as multihashes from 'multihashes';
-import {bases} from 'multiformats/basics';
+import { inappropriateMultiaddr, invalidArgument, invalidFingerprint, unsupportedHashAlgorithm } from './error'
+import { logger } from '@libp2p/logger'
+import { Multiaddr } from '@multiformats/multiaddr'
+import * as multihashes from 'multihashes'
+import { bases } from 'multiformats/basics'
 
-const log = logger('libp2p:webrtc:sdp');
+const log = logger('libp2p:webrtc:sdp')
+const CERTHASH_CODE: number = 466
 
+// Get base2 | identity decoders
 export const mbdecoder: any = (function () {
-  const decoders = Object.values(bases).map((b) => b.decoder);
-  let acc = decoders[0].or(decoders[1]);
-  decoders.slice(2).forEach((d) => (acc = acc.or(d)));
-  return acc;
-})();
+  const decoders = Object.values(bases).map((b) => b.decoder)
+  let acc = decoders[0].or(decoders[1])
+  decoders.slice(2).forEach((d) => (acc = acc.or(d)))
+  return acc
+})()
 
-const CERTHASH_CODE: number = 466;
-
-function ipv(ma: Multiaddr): string {
+// Extract the ipv from a multiaddr
```

Comment:
imho, a decent name would make comments like this not needed (e.g. `function extractIpv(...)`

Reply:
Good point.  Maybe `ma2ipv` would fit the other naming conventions?

Reply:
very cryptic though :P but does fit indeed

Reply:
I like these comments!  The next PR will focus more on documentation, readability, maintainability, and testability, so hope to keep this focus on that PR when it's ready.  We'll want that one to be very polished.